presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcBatchPageSource.java

```diff
@@ -74,6 +79,19 @@ public OrcBatchPageSource(
             OrcAggregatedMemoryContext systemMemoryContext,
             FileFormatDataSourceStats stats,
             RuntimeStats runtimeStats)
+    {
+        this(recordReader, orcDataSource, columns, typeManager, systemMemoryContext, stats, runtimeStats, nCopies(columns.size(), false));
+    }
+
+    public OrcBatchPageSource(
```

Comment:
Can you create an issue to add whatever is necessary for the corresponding selective page source?